h/static/scripts/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "checkJs": true,
+    "lib": ["es2018", "dom"],
+    "jsx": "react-jsx",
+    "jsxImportSource": "preact",
+    "module": "commonjs",
```

Comment:
The `module` configuration differs across repos. We probably should be using `es2020` everywhere, which is the configuration in the client. If making that change doesn't cause any errors, I'd do it now, otherwise we can revisit later.

Reply:
Done: https://github.com/hypothesis/h/pull/8789/commits/f7f6b97653b9bc01d6e5cc7e65bfa692cce7e964